Content.Client/Chat/Managers/ChatManager.cs

```diff
@@ -12,26 +15,20 @@ internal sealed class ChatManager : IChatManager
     [Dependency] private readonly IClientConsoleHost _consoleHost = default!;
     [Dependency] private readonly IClientAdminManager _adminMgr = default!;
     [Dependency] private readonly IEntitySystemManager _systems = default!;
+    [Dependency] private readonly IPrototypeManager _prototypeManager = default!;
 
     private ISawmill _sawmill = default!;
 
+    [ValidatePrototypeId<RadioChannelPrototype>]
+    private const string DefaultRadioChannel = "Common";
+
     public void Initialize()
     {
         _sawmill = Logger.GetSawmill("chat");
         _sawmill.Level = LogLevel.Info;
     }
 
-    public void SendAdminAlert(string message)
-    {
-        // See server-side manager. This just exists for shared code.
-    }
-
-    public void SendAdminAlert(EntityUid player, string message)
-    {
-        // See server-side manager. This just exists for shared code.
-    }
-
-    public void SendMessage(string text, ChatSelectChannel channel)
+    public void SendMessage(string text, ChatSelectChannel channel, RadioChannelPrototype? radioChannel)
     {
         var str = text.ToString();
         switch (channel)
```

Comment:
this looks like old-old code, is it left there just for backwards compatibility? If so i'd like to propos Proxy or Adapter design pattern here <_< low priority thing ofc...

Reply:
maybe a little of todo with a link to separate issue for cleanup? even if issue will be placeholder - u would be able to fill it up later :godo: